knife/knife.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |s|
   s.add_dependency "mixlib-cli", ">= 2.1.1", "< 3.0"
   s.add_dependency "mixlib-archive", ">= 0.4", "< 2.0"
   s.add_dependency "ohai", "~> 17.0"
-  s.add_dependency "ffi", "~> 1.15.5" # 1.14 versions are broken on i386 windows
+  s.add_dependency "ffi", "~> 1.16" # 1.14 versions are broken on i386 windows
```

Comment:
Use `~> 1.16.3` to box it to >= 1.16.0 and < 1.17

Reply:
gotcha